pygfx/objects/_events.py

```diff
@@ -118,13 +137,63 @@ def _update_current_target(self, target):
 
 
 class KeyboardEvent(Event):
+    """Keyboard button press.
+
+    Parameters
+    ----------
+    args : Any
+        Positional arguments are forwarded to the :class:`base class
+        <pygfx.objects.Event>`.
+    key : str
+        The key that was pressed.
+    modifiers : list
+        The modifiers that were pressed while the key was pressed.
+    kwargs : Any
+        Additional keyword arguments are forward to the :class:`base class
+        <pygfx.objects.Event>`.
+
+    """
+
     def __init__(self, *args, key, modifiers=None, **kwargs):
         super().__init__(*args, **kwargs)
         self.key = key
         self.modifiers = modifiers or ()
 
 
 class PointerEvent(Event):
+    """Mouse/Touch Event.
+
+    Parameters
+    ----------
+    args : Any
+        Positional arguments are forwarded to the :class:`base class
+        <pygfx.objects.Event>`.
+    x : int
+        The x position of the cursor or touch in screen space (px).
+    y : int
+        Thy y position of the cursor or touch in screen space (px).
+    button : int
+        @almarklein: where can I find the description of this attribute?
```

Comment:
This is the spec that we follow for values like these: https://jupyter-rfb.readthedocs.io/en/latest/events.html

Reply:
I'll add the link to the docstring.